_posts/2017-01-23-upgrade-guide-for-babel-7-authors.md

```diff
@@ -23,6 +23,44 @@ This document is for developers that create tools that depend on Babel.
 
 This had to be used to prevent a breaking change with our exports. Now if you import a babel package in a library you may need to use `.default` for commonjs.
 
+## Babel Presets
+
+> Dropped support for exporting as an object [#5128](https://github.com/babel/babel/pull/5128)
+
+A preset needs to export a function as the default export or with module.exports.
+
+Babel 6
+
+```js
+module.exports = {
+  buildPreset: function () {
+    return {
+      plugins: ['plugin-a']
+    };
+  }
+};
+
+export default {
+  plugins: ['plugin-a']
+}
+```
+
+Babel 7
+
+```js
+module.exports = function () {
```

Comment:
I took a while to parse this. Could we separate theses blocks or/and specifiy CJS or ES6 Module?